### ISSUE TITLE
Context Service: `routerContext` always needs to be true to update the page title

### ISSUE THREAD

**Issue by lievenvg** [digipolisantwerp/acpaas-ui_angular#72]
## Describe the bug

The `routerContext` boolean of the Context Service always needs to be set with the value `true` to update the page title.
This should not be needed, because this creates an extra listener on routechange.

## To Reproduce

1. Add the ContextModule to your appModule

```typescript
import { CookieconsentModule } from '@acpaas-ui/ngx-components/layout';

ContextModule.forRoot({
   defaults: {
      title: 'Context title',
   },
   routerContext: true,
}),
```

## Expected behavior

`routerContext` shouldn't be needed.

## OS (please complete the following information):

 - Device: MacBook Pro (Retina, 13-inch, Late 2013)
 - OS: macOS High Sierra (Version 10.13.6)

**Comment by gerturo** [comment]
Picked up by ACPaaS frontend team

**Comment by lievenvg** [comment]
Solved in #123
